src/witness/witness_calculator.rs

```diff
@@ -16,8 +16,10 @@ use super::Circom;
 
 #[derive(Clone, Debug)]
 pub struct WitnessCalculator {
-    pub instance: Wasm,
+    pub store: Arc<RwLock<Store>>,
```

Comment:
`store` here is public, whereas before it was internal to the struct. Is this intended?

Reply:
Nope, will change to private